### ISSUE TITLE
[RPA.Email.ImapSmtp] "Send message" with attachments using relative path statements does not work (#148 related)

### ISSUE THREAD

**Issue by GSymonow** [robocorp/rpaframework#160]
Hello,
I 've retested  #148  issue and spotted problems.

I'm using relative paths to attachments like:
-  ${CURDIR}${/}..${/}..${/}test_data${/}attachments${/}${strAttachment} or
- .${/}test_data${/}attachments${/}${strAttachment}

After #148 fix in both cases stated above the "Send Message" results with such error:
"OSError: [Errno 22] Invalid argument: "['Normalize Path C:\\\\Users\\\\suser\\\\PycharmProjects\\\\ProjectRobotFramework\\\\resources\\\\Keywords\\\\..\\\\..\\\\test_data\\\\attachments\\\\test.xml']""

It seems that the statements like "." (execution dir) or ".." (go to parent dir) are ignored and treated like a plain text.
Could you please check that.
Maybe there is something wrong on my side but I've checked several conditions and for me it looks like an error.
Thanks in advance.
Best Regards,
Greg

**Comment by mikahanninen** [comment]
@GSymonow Hi. Can you show in a bit more detail how you are sending attachments ?

For me following worked on MacOS (I will check the Windows tomorrow)
```
    Send Message    ${sender}    ${receiver}
    ...    subject=incoming invoice
    ...    body=incoming invoice
    ...    attachments=.${/}..${/}..${/}..${/}code.png
```

**Comment by GSymonow** [comment]
@mikahanninen thanks for taking care the issue.

I think I found the root cause for this.
I was passing the 'attachments' string for the 'Send Message' by the @{strAttachments} list variable in the keyword definition.
```
  Send Message    ${sender}    ${receiver}
    ...    subject=incoming invoice
    ...    body=incoming invoice
    ...    attachments=@{strAttachments}
```
I wanted to have the flexible keyword with ability to send one or multiple attachments so that was the reason that I used the List.
After changing it to standard variable it works fine.

```
  Send Message    ${sender}    ${receiver}
    ...    subject=incoming invoice
    ...    body=incoming invoice
    ...    attachments=${strAttachments}
```

It worked fine with the Lists for the earlier version than 8.0.0. After upgrade I spotted the issue.
Still I'm not sure if this is an issue with your 'Send Message' keyword or maybe some general changes with Lists support.
